scripts/check-pkg-for-release.js

```diff
@@ -256,11 +254,15 @@ function checkAutoInitAddedInMDCPackage(ast) {
       const callee = node.expression.callee;
       const args = node.expression.arguments;
       if (callee.object.name === 'autoInit' && callee.property.name === 'register') {
-        const expression = args.find((value) => {
-          return value.type === 'MemberExpression';
-        });
-        if (expression.object.name === nameCamel) {
-          autoInitedCount++;
+        for (let value of args) {
+          while (value.type === 'TSAsExpression') {
+            value = value.expression;
+          }
+
+          if (value.type === 'MemberExpression' && value.object.name === nameCamel) {
```

Comment:
Can you add a comment explaining why we check for the string "MemberExpression"?

Reply:
Done